docs/maven-tutorial.md

```diff
@@ -0,0 +1,218 @@
+# OpenShift + Maven Tutorial
+
+While developing your application with Java, you'll most likely be building
+many, many, many times.  In order to shorten the build times of your pods,
+dependencies can be cached in local Maven repositories.  This tutorial will step
```

Comment:
It confused me a bit, because "local Maven repository" is `~/.m2/repository` directory. The explanation is also valid for this case, so it's unclear what the benefits Nexus will give me.